docs/referenceConf.js

```diff
@@ -197,6 +197,10 @@ exports.config = {
   // capability.
   // You can specify a file containing code to run by setting onPrepare to
   // the filename string.
+  // If the preparation involves any asynchronous calls (e.g. interacting with
+  // the browser), it should return a promise. Otherwise Protractor will be
```

Comment:
Maybe something along this line:
OnPrepare can optionally return a promise, which Protractor will wait for before continuing execution. If the preparation involves any asynchronous calls (e.g. interacting with the browser), notify Protractor to wait by returning a promise.